install/connector/bin/ts.jte

```diff
@@ -325,12 +325,11 @@ command.testExecute=com.sun.ts.lib.harness.ExecTSTestCmd \
         -Djavax.net.ssl.trustStore=${s1as.domain}/${sjsas.instance.config.dir}/cacerts.jks \
         -Dcom.sun.aas.installRoot=${connector.home} \
         -Dcom.sun.aas.configRoot=${connector.home}/config \
-        -Djava.util.logging.config.file=${bin.dir}/logging.properties \
```

Comment:
https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/1944/execution/node/302/log/?consoleFull
It seems this property is not used at all.
Locally the connector TCK passed for the current GF master.

Reply:
It is really hard to trace all paths how the file and the property could be used, so I would just try it as a simplest way to detect if there are further steps required or if this PR is completely fine.
The original state wasn't correct.

Reply:
I think this file will not be used for platform TCK run. If the connector standalone TCK does not have the same issue it should be ok to remove the changes from this file and install/j2eemgmt/bin/ts.jte